api/api.go

```diff
@@ -271,16 +271,21 @@ type OrbConfigResponse struct {
 // OrbCollection is a container type for multiple orbs to share formatting
 // functions on them.
 type OrbCollection struct {
-	Orbs      []Orb  `json:"orbs"`
-	Namespace string `json:"namespace,omitempty"`
+	Orbs          []Orb  `json:"orbs"`
+	Namespace     string `json:"namespace,omitempty"`
+	PrintDetailed bool   `json:"-"`
```

Comment:
It seems strange that the `api` package needs to be informed of how the results will be printed. This should only be a concern of the `cmd` package.

Since all of the `OrbCollection` fields are exported I think this could be accomplished by removing `String` and moving the implementation into the `cmd` package.